diesel/src/macros/mod.rs

```diff
@@ -928,13 +928,13 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// * `child_table` is the Table with the Foreign key.
 ///
-/// So given the Table decaration from [Associations docs](http://docs.diesel.rs/diesel/associations/index.html)
+/// So given the Table decaration from [Associations docs](https://docs.diesel.rs/diesel/associations/index.html)
```

Comment:
Those link should rather be relative instead of absolute

```suggestion
/// So given the Table decaration from [Associations docs](associations/index.html)
```

Reply:
Thank you, I changed these.